Sources/Apodini/Properties/ObservableObject.swift

```diff
@@ -0,0 +1,6 @@
+/// Objects that notify subscribers when `@Published` properties change.
+///
+/// Refer to the documentation of
+/// [ObservedObject](https://github.com/Apodini/Apodini/blob/develop/Documentation/Communicational%20Patterns/2.%20Tooling/2.4.%20ObservedObject.md)
```

Comment:
Please add more docs here in code. I don't think we should cross reference our textual documentation like this as such references will break most likely in the future.

Reply:
Changed the documentation